README.md

```diff
@@ -1,64 +1,59 @@
+[Live Example](https://8caf94bb-c2e5-45cf-898d-ca5b19954cda.api.master.kintohub.com/express-example/hello/wolrd)
 
-## Requirements
+__About KintoHub:__
 
-- Ensure you have an account on [KintoHub](www.kintohub.com/sign-up)
-- TODO: Any pre-reqs to list here
+KintoHub aligns teams to ship & operate cloud native apps with ease. [Learn More](https://www.kintohub.com)
 
 ## Deployment
-1. Apply this template to your [Github](https://github.com/kintohub/example-template/generate)
-2. Create a [KintoBlockType](www.link.to.create.kintoblock.type.here)
-3. Instructions on what to input in order to create KintoBlock
- 
-[Optional]
- ?. Click on "Build Arguments" and paste the following build args
-```dotenv
-key1=value
-key2=value
-```
-[Optional]
- ?. Click on "Settings" at the top left and paste the following into Environment Variables
-```dotenv
- key1=value
- key2=value
-```
+1. Apply this template to your [Github](https://github.com/kintohub/express-example/generate)
+2. Create a [Microservice Block](https://docs.kintohub.com/docs/kintoblocks/microservices) on KintoHub
+3. Connect your GithubApp to KintoHub and select the Repository you just generated on your account.
+4. Set the **name** of your kinto block
+5. Choose `Node.js` as the **language** and `11.7.0` as the **version**
+6. Set the **Build Command** as `npm install`
+7. Set the **Start Command** as `npm start`
+8. Set the **Port** as `80`
 
-You're now good to go! Click `Build Latest Commit` and once successful `Add To Project` to deploy.
+You're now good to go! Click **Create Microservice**. Now click **Build Latest Commit**
 
-### TODO:(Optional) Advance Configuration For Production
+... The build takes about 40 seconds. Once complete, Click *Add to Project*
 
-The above instructions are great for `dev` but if you want to setup `production` configuration, we recommend the following.
+Follow the instruction to creating a new project. Once created, you can then click *Open* on your Ionic block and see it running live!
 
-TODO: Provide Instructions via Build Args and/or Kinto Configuration for production use. Ensure each explain tells why it makes it production ready (in one sentence - dont do lengthy!)
+## Installation & Local Run
+Ensure you have node 10 or higher.
 
+1. `npm install`
+2. `npm start`
 
 ## Usage
+#### Optional Environment Variables
+Default value  = `Hello`, please refer to the `.env-example` file
+```
+GREETING=Hi
+```
 
-1. Provide
-2. Clear
-3. Instructions
-4. On
-5. How
-6. To
-7. Use or Access
+#### Endpoint
+[Live Example](https://8caf94bb-c2e5-45cf-898d-ca5b19954cda.api.master.kintohub.com/express-example/hello/wolrd)
+```
+GET http://localhost/hello/world
```

Comment:
I'm not sure why this comment didnt go through... I wrote a comment on this URL will not work. should make it more clear that its based on the kintoblock name.

Weird - really wondering if github just lost some data?